forest/shared/src/cli/snapshot_fetch.rs

```diff
@@ -56,20 +82,30 @@ pub fn is_aria2_installed() -> bool {
 /// Fetches snapshot for `calibnet` from a default, trusted location. On success, the snapshot will be
 /// saved in the given directory. In case of failure (e.g. connection interrupted) it will not be
 /// removed.
-async fn snapshot_fetch_calibnet(
+async fn snapshot_fetch_forest(
     snapshot_out_dir: &Path,
-    snapshot_fetch_config: &SnapshotFetchConfig,
+    config: &Config,
     use_aria2: bool,
+    compressed: bool,
+    skip_checksum_validation: bool,
 ) -> anyhow::Result<PathBuf> {
-    let name = &snapshot_fetch_config.calibnet.bucket_name;
-    let region = &snapshot_fetch_config.calibnet.region;
+    if compressed {
+        bail!("Compressed snapshot is not available, Use `--filops` flag to download compressed snapshot from filops servers.");
+    }
```

Comment:
Let's not support compressed snapshots until we host them ourselves.

Reply:
okay